src/shared/varlink-io.systemd.Unit.c

```diff
@@ -157,13 +157,81 @@ static SD_VARLINK_DEFINE_STRUCT_TYPE(
                 SD_VARLINK_FIELD_COMMENT("Whether this unit is perpetual"),
                 SD_VARLINK_DEFINE_FIELD(Perpetual, SD_VARLINK_BOOL, 0));
 
+static SD_VARLINK_DEFINE_STRUCT_TYPE(
+                ActivationDetail,
```

Comment:
Let's name this ActivationDetails like the internal struct. "Details" on the activation is still singular if you ask me.